### DISCUSSION TITLE
so is this working at all?

### DISCUSSION THREAD

**Discussion by ravarkon** [Matt-FTW/dotfiles#18] [Q&A]
so it's not a bug or anything like that, how ever after trying many times i don't get passed the login area haha

so using archinstaller, and i ran everything but i get few errors which i try to solve - such as linux-headers missing, and then the hyprpm update does not update, then missing hypr-bar???? then the Super+D does noting as well so not sure if there was another way around this?? or i have to install openssh? and try that way again or is there something am missing when installing this?

**Comment by Matt-FTW** [comment]
Hi,

Sorry about it! Let me explain:

First of all, the linux-headers package is one that you have install in the arch installation. Im not going to include it because is basically the base of the system, which I asume everyone trying to install the dotfiles has.

About the hyprprm update issue. It has to be an issue related to the hyprland version or hyprland itself. It breaks a lot of times once a new update drops, and my dotfiles cannot influence anything.

When you said hypr-bar, did you ment Waybar? In that case, is on the installation instructions.

The Super+D binding opens up a new scratchpad with lazygit. if you dont have lazygit, its not going to launch. I didnt include lazygit because its a more specific package and I expect people that want to use it install it by themselves.

Finally, if you're getting some issues with the login, can you post the error output? If you are using sddm or some other display manager, open a new terminal space and launch hyprland with the Hyprland command itself, and see what it tells you.

**Reply by ravarkon** [reply]
ah very fast in replay :D i feel so much special :) anyhow there no post output due to i been wipeing my drive out about 4 times now hah just to make it work, as for the waybar, yeah it installed. but does not load up i even re-install it and still does not show up.. so i wonder as u said something must of got updated and broke something long the line.. as for the lazygit i will try that haha. 

i will update you shorty and great job on the dotfiles its just right :)

**Reply by Matt-FTW** [reply]
4 times? lmao. I mean, I dont know the archinstall its now, but at the time I tried it was really good and did not have any problems. 

Know I understand the waybar problem. You have to change to output value to yours. I didnt put it on the installation instructions :p...

Next time, if the waybar is not working, be sure to launch a terminal and execute waybar on there. It will tell you whats the issue.

btw, thx for your kind words :)

**Reply by ravarkon** [reply]
yeah 4 times haha it's sad i know, how ever am use to ISO more when comes to arch install.. but i will get there soon 🗡️ 
and yep i will let u know what the output will be onces its does installing the 5th time :)

**Reply by Matt-FTW** [reply]
good luck buddy :crossed_fingers:

**Reply by ravarkon** [reply]
haha i got the hyprpm working with the plugins, now i been trying to get the waybar to load..

when i run the "waybar" it only shows up [info] debug, but noting more unless i do "ctrl+c" to close it.. but Does not show up on the desktop.. maybe i do a reinstall waybar it can maybe show up?

**Reply by Matt-FTW** [reply]
Thats because you have to change the output definition on the [config.jsonc file](https://github.com/Matt-FTW/dotfiles/blob/aff284ce8ad3eba9969316f001a781d878cda1db/.config/waybar/config.jsonc#L6).

**Reply by ravarkon** [reply]
Yep i fix that haha so waybar is working good, just working on the keybord lang, as well the weather not showing up yet.. so i did change the area of the weather so just waiting.. also is there a command for reload - restart waybar (question mark lol i can´t use question mark due to KeyBord using ES ahha

**Reply by Matt-FTW** [reply]
Nice, glad you fixed it. Check [this page in the docs](https://wiki.hyprland.org/Configuring/Uncommon-tips--tricks/#switchable-keyboard-layouts) if you wanna know more about changing the layout. And [this file](https://github.com/Matt-FTW/dotfiles/blob/main/.config/hypr/configs/input.conf) where I have it defined. Its weird tho, should select US by default.

The keybinding to reload waybar is Ctrl + Win + R. Check [the binds file](https://github.com/Matt-FTW/dotfiles/blob/main/.config/hypr/configs/binds.conf) next time. And to change the layout, Win + Space.

**Reply by ravarkon** [reply]
Yep that fix that issue haha. now working on the weather and the display mode..and then it should be 100% done haha...

i should backup these configs for this dell haha older dell that is..

**Reply by ravarkon** [reply]
i did a 7th reinstall on arch and now it's all working as it needs other then weather still.. lol not even sure why its not showing up... hmmm TIME TO DIG DEEP! also u have dc? (discord)?

**Reply by Matt-FTW** [reply]
Lets gooo :) 

For the weather you have to change the city in [this line](https://github.com/Matt-FTW/dotfiles/blob/ce4677ac2ce47101f6359f0332c0dae679bf92af/.config/waybar/modules/custom-weather.jsonc#L4) to yours. Be sure to have [wttrbar](https://github.com/bjesus/wttrbar) in your $PATH. Also, it uses the [wttr.in API](https://wttr.in/), if you wanna check it out to learn more about it.

Yeah I have a discord. Send me yours at my email (you can find it on my profile), that way it doesnt leak over here.
